lib/Server.js

```diff
@@ -91,6 +82,15 @@ class Server {
       throw new Error("'filename' option must be set in lazy mode.");
     }
 
+    this.stats = options.stats;
+    this.DEFAULT_STATS = {
+      all: false,
+      hash: true,
+      assets: true,
+      warnings: true,
+      errors: true,
+      errorDetails: false,
+    };
```

Comment:
Let's avoid `this.DEFAULT_STATS` (because it should be as private property)

Reply:
I needed it for the tests.
Should I add them to the Server's prototype then?

Reply:
Can you provide link where you need it for test? You already have `getStats` method

Reply:
[this is where I'm using it](https://github.com/webpack/webpack-dev-server/pull/1665/files#diff-7775e181de5f32a001a1636103e7008fR33)